SYCL/Basic/group_async_copy.cpp

```diff
@@ -24,14 +28,14 @@ template <typename T> void initInputBuffer(buffer<T, 1> &Buf, size_t Stride) {
   auto Acc = Buf.template get_access<access::mode::write>();
   for (size_t I = 0; I < Buf.get_count(); I += WorkGroupSize) {
     for (size_t J = 0; J < WorkGroupSize; J++)
-      Acc[I + J] = I + J + ((J % Stride == 0) ? 100 : 0);
+      Acc[I + J] = (T)(I + J + ((J % Stride == 0) ? 100 : 0));
```

Comment:
Is there any reason for using C-style cast?

Reply:
I replaced it with static_cast